plugin/tagbar.vim

```diff
@@ -101,6 +101,7 @@ function! s:setup_options() abort
         \ ['position', default_pos],
         \ ['previewwin_pos', previewwin_pos],
         \ ['scopestrs', {}],
+        \ ['scrolloff', 0],
```

Comment:
I don't use scrolloff much myself (except when I set it to 999 to keey the current line centered for some operations) so I don't know if it would make any sense, but is there any logic in setting the default to the parent buffer's setting of `scrolloff`?

Reply:
that's a good idea.... done

Reply:
Hmm... actually on second thought we shouldn't need to do this.

By setting a default of `0`, then we do not even issue the `setlocal scrolloff=#` so the global `scrolloff` value will be honored.

Reply:
In that case that's better, only set this if you don't want the inherited value. :+1: